client/components/timezone-dropdown/docs/example.jsx

```diff
@@ -34,7 +34,7 @@ export default React.createClass( {
 					<a href="/devdocs/design/timezone-dropdown">TimezoneDropdown</a>
 				</h2>
 
-				<Card style={ { width: '300px', margin: 0 } }>
+				<Card style={ { width: '300px', height: '350px', margin: 0 } }>
```

Comment:
I think it should be handled directly in devdocs because the component is mounted pretty well.

![image](https://cloud.githubusercontent.com/assets/77539/13966140/d7fb8c50-f04f-11e5-822e-2586334e55b2.png)